backend/app/tests/api/routes/test_test.py

```
@@ -2414,4 +2414,319 @@ def test_get_inactive_tests_not_listed(
     data = response.json()
 
     assert response.status_code == 200
-    assert all(item["id"] != test_id for item in data)
+
+    assert all(item["id"] != test.id for item in data)
```

Ruff (0.11.9)

Undefined name `test`

(F821)